### DISCUSSION TITLE
Getting DXGI_ERROR_DEVICE_REMOVED crash while playing.

### DISCUSSION THREAD

**Discussion by LoreAPC** [seerge/g-helper#2740] [Q&A]
While playing Redout 2 and sometimes with Baldur's gate 3 I randomly get this crash:
![20240616_172438](https://github.com/seerge/g-helper/assets/128968377/588e0b77-a4b2-490b-89b7-f6d16d76330e)

Remove devices sounds like something ghelper might have someting to do, so I thought of asking here...

Some info:
Zephyrus G14 2021 RTX 3060
Ghelper 0.176 (no AC)
Balanced profile, no core/vram undervolt
No power relatet settings, all default (cpu boost disabled)
Playing on external monitor, connected on HDMI (no mux present)

GPU temps in the middle 60s while playing.
GPU driver updated

Does anybody have some suggestions, besides obvious "reinstall windows"?

Thanks!

**Comment by seerge** [comment]
@LoreAPC hello, G-Helper doesn't run your hardware in realtime anyhow, so I don't think it's anyhow related. 

Quick googling has led me to info that it's a typical DirectX (gaming library / SDK) error 
https://www.makeuseof.com/dxgi-error-device-removed-error-windows/ 

It seem to be a post on reddit with someone's advice on possible fixes
https://www.reddit.com/r/ARK/comments/17plsn4/possible_fix_for_dxgi_error_device_removed_with/

**Comment by LoreAPC** [comment]
Yeah I've seen those posts but also other similar posts saying it doesn't work...I've seen a fix "mostly" working which is using Afterburner to underclock the GPU...do you think it is possible with this notebook?
I also tried overclocking with Ghelper (+170/+170) which should also undervolt but the error persists...

Do you think it's a software issue, then? Not hardware?
Thanks for your answer!

**Reply by seerge** [reply]
@LoreAPC you can try to underclock, buy setting negative Core / Memory clock offsets. That would make GPU run on a bit lower clocks with same (old) power consumption. If there are some hardware issues - this can theoretically help sometimes, i think :)

**Comment by LoreAPC** [comment]
Thanks, I'll also try what you suggest on the undervolting section of the
wiki, which is to overclock and limit the core clock.
I'll use Redout2 as my benchmark since it is the game that crashes the most.

Another question, if I may ask: does Afterburner provide more controls and
function than ghelper?

Il Dom 16 Giu 2024, 18:55 Serge ***@***.***> ha scritto:

**Reply by seerge** [reply]
@LoreAPC i actually suggested to do opposite - to "overvolt" / underclock (assuming you suspect stability issues in games) :)  

You need to drag Core Clock / Memory offset sliders to the left, to a negative values. That will lower clocks / performance but may help with stability.

Afterburner offers similar controls, both apps use Nvidia API to set same things.

**Comment by LoreAPC** [comment]
I'll try both ways!
1) Overclock/undervolt+clock limit to decrease power consumption and
possibly temperature
2) Underclock/overvolt to decrease clocks

Let's see if something works :)

In the meantime, thank you very much for your answers!

Il Lun 17 Giu 2024, 10:27 Serge ***@***.***> ha scritto:

**Comment by LoreAPC** [comment]
Hi! In the end I tried varoius settings in Ghelper and also tried adding the lines in the registry but Redout 2 keeps crashing...the only difference is that now the game freezes for one minute before crashing thanks to the timer added with the registry tweak...
Instead I played a bungh of hours in BG3 also with overclocked GPU core/memory (+170 / +170) and had no crashes...

I guess it's a problem with specific games...
So thank you very much for your assistance and support!

**Comment by LifeKife** [comment]
I had the same problem on the RTX4070TI video card. I tried everything, but nothing helped. I found a post on the Internet that said that lowering the GPU frequency by 50 MHz helps. I started running the video card in stress tests and the temperature was normal, but I still found one nuance. In FurMark, the HOT SPOT reached 110 degrees at a GPU temperature of 80 degrees. And, apparently, this was the problem. The video card goes into hot spot protection and turns off. Moreover, the cooler speed is regulated based on the overall GPU temperature, and the GPU is turned off when it overheats according to the hot spot. If you have encountered this problem, check the hot spot in FurMark, if the difference between the GPU temperature and the hot spot is higher than 20 degrees, you are guaranteed to have video card glitches. Summary. Check the hot spot for overheating, it should not be higher than 90 degrees. Above 110, throttling occurs and an emergency shutdown is possible.

How to solve the problem if you really have overheating on a hot spot, you need to replace the GPU thermal paste with a new one with high thermal conductivity
